### DISCUSSION TITLE
Preview of document shows exact page of search results

### DISCUSSION THREAD

**Discussion by ShilpaSharePoint** [microsoft-search/pnp-modern-search#2477] [Q&A]
Hi, as a part of our search solution we won't show the preview of document (document with multiple pages) with exact page where the search results is available. Any pointers on this will be helpful. We are using PNP Modern Search solution as a base & uses Graph API for search not the SharePoint API.

**Comment by wobba** [comment]
I don't quite understand the issue. Can you explain better what you see, and what you would like to change? What is the web part setup?

**Reply by ShilpaSharePoint** [reply]
Hi, Part of Search Results webpart, when we place cursor over any search results particularly document results, when we click on those results, it should navigate to exact page (like if document has 10 pages, search result in 7th page, it should navigate to 7th page) instead just opening the page.

**Reply by wobba** [reply]
That's not possible as page numbers are not part of search result hits, and client applications don't support linking directly to specific pages at the moment. A better way is when client apps support a parameter with the search terms, and then does on-page search when you open.

**Reply by ShilpaSharePoint** [reply]
Thanks for the reply
